src/Microsoft.ML.Core/Utilities/FloatUtils.cs

```diff
@@ -438,14 +438,11 @@ public static bool IsFinite(Single x)
             return bits.IsFinite();
         }
 
-        public static bool IsFinite(Double[] values, int count)
+        public static bool IsFinite(ReadOnlySpan<Double> values)
```

Comment:
Would `double` be better?
```suggestion
        public static bool IsFinite(ReadOnlySpan<double> values)
```

Reply:
fixed the whole file. thanks